### DISCUSSION TITLE
./infra/example-configuration.sh

### DISCUSSION THREAD

**Discussion by shivarm** [Hyperfoil/Horreum#590] [General]
What does mean of this?

```
./infra/example-configuration.sh
POST /schema
Name already used
##########################
#      IMPORT FAILED     #
##########################

**Comment by jesperpedersen** [comment]
Try and remove the `horreum_horreum_pg13` volume

**Comment by johnaohara** [comment]
I am currently working on a dev service for horreum that will remove the '/infra' dir and manage the development env via `mvn quarkus:dev`. You can track it in this branch: https://github.com/johnaohara/Horreum/commits/horreum-dev-services
